starknet_py/tests/e2e/docs/guide/test_cairo1_contract.py

```diff
@@ -13,23 +12,17 @@
 async def test_cairo1_contract(
     account,
     sierra_minimal_compiled_contract_and_class_hash,
-    another_sierra_minimal_compiled_contract_and_class_hash,
     gateway_client,
 ):
-    # pylint: disable=import-outside-toplevel, too-many-locals
+    # pylint: disable=too-many-locals
+    # pylint: disable=import-outside-toplevel
     (
         compiled_contract,
         compiled_class_hash,
-    ) = (
-        sierra_minimal_compiled_contract_and_class_hash
-        if isinstance(account.client, GatewayClient)
-        else another_sierra_minimal_compiled_contract_and_class_hash
-    )
+    ) = sierra_minimal_compiled_contract_and_class_hash
 
     contract_compiled_casm = read_contract(
-        "minimal_contract_compiled.casm"
-        if isinstance(account.client, GatewayClient)
-        else "another_minimal_contract_compiled.casm",
+        "minimal_contract_compiled.casm",
         directory=CONTRACTS_COMPILED_V1_DIR,
     )
```

Comment:
I was wondering for a second how come this works, but realised we're running full node and gateway separately 👍